static/js/teams.js

```diff
@@ -0,0 +1,38 @@
+const teams = JSON.parse(document.getElementById('teams').textContent);
+const leftover_participants = JSON.parse(document.getElementById('leftover_participants').textContent);
+
+
+function allowDrop(ev) {
+    ev.preventDefault();
+  }
+  
+function drag(ev) {
+    ev.dataTransfer.setData("text", ev.target.id);
+}
+
+function drop(ev) {
+    let targetElement;
+    ev.preventDefault();
+    var data = ev.dataTransfer.getData("text");
+    if(ev.target.nodeName == 'LI') {
+        targetElement = ev.target.parentElement.id;
+        ev.target.parentElement.appendChild(document.getElementById(data));
+    } else {
+        targetElement = ev.target.id;
+        ev.target.appendChild(document.getElementById(data));
+    }
+    moveTeamData(data, targetElement)
+}
+
+function moveTeamData(movedElement, targetElement){
```

Comment:
Can we get some code-comments on what these functions are intended to do? It looks/appears as if the admin/superuser is able to drag/drop participants between different teams perhaps? I don't see any `canvas` for draggable elements though?

Reply:
Yes, exactly. Added a comment. Not sure I understand the reference to the `canvas`, I followed the w3schools drag & drop api here:https://www.w3schools.com/html/html5_draganddrop.asp